### PR TITLE
Gui 2759 Security policy dialog integrated into wizard

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2759

### CODE REVIEW

File: eucaconsole/static/js/widgets/listener-editor/security-policy-editor.js [hunk 1]
```diff
@@ -1,12 +1,118 @@
...
+        controller: ['$scope', '$timeout', 'ModalService', 'ELBService', 'eucaHandleError', function ($scope, $timeout, ModalService, ELBService, eucaHandleError) {
+            var vm = this;
+            vm.policyRadioButton = 'existing';
+            vm.predefinedPolicyChoices = [];
+            vm.protocolChoices = [
+                {id:'Protocol-TLSv1.2', label:'TLSv1.2'},
+                {id:'Protocol-TLSv1.1', label:'TLSv1.1'},
+                {id:'Protocol-TLSv1', label:'TLSv1'},
+            ];
+            vm.cipherChoices = [
```
Comment (kamalgill) [inline eucaconsole/static/js/widgets/listener-editor/security-policy-editor.js]: @dkavanagh: Why not leverage the existing SSL_CIPHERS list in constants/elbs.py?

File: eucaconsole/templates/elbs/listener-editor/security-policy.pt [hunk 2]
```diff
@@ -1,38 +1,92 @@
...
+                        </label>
+                    </div>
+                    <div class="large-12 small-12 columns field">
+                        <select id="ssl_protocols" required="required" name="ssl_protocols"
+                                ng-model="policy.sslProtocols" chosen="true"
+                                ng-init="policy.sslProtocols = policyCtrl.protocolChoices.slice()"
+                                ng-options="opt as opt.label for opt in policyCtrl.protocolChoices track by opt.id"
+                                multiple="" data-placeholder="Select..." i18n:attributes="data-placeholder">
+                        </select>
+                        <small class="error">At least one protocol is required.</small>
```
Comment (kamalgill) [inline eucaconsole/templates/elbs/listener-editor/security-policy.pt]: @dkavanagh : Need to mark this for i18n

File: eucaconsole/static/js/widgets/listener-editor/security-policy-editor.js (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/static/js/widgets/listener-editor/security-policy-editor.js]: I stand corrected. That form can be removed, but the constant is used in the form below it. Keep that for now.

File: eucaconsole/forms/__init__.py [hunk 3]
```diff
@@ -384,19 +385,9 @@ def predefined_policy_choices(self, add_blank=True):
...
-            root = ElementTree.fromstring(resp.read())
-            policy_descriptions = root.find('.//{0}PolicyDescriptions'.format(xml_prefix))
-            policies = policy_descriptions.getchildren() if policy_descriptions is not None else []
-            for policy in policies:
-                policy_type = policy.find('.//{0}PolicyTypeName'.format(xml_prefix))
-                if policy_type is not None and policy_type.text == 'SSLNegotiationPolicyType':
-                    policy_name = policy.find('.//{0}PolicyName'.format(xml_prefix)).text
-                    if policy_name.startswith(ELB_PREDEFINED_SECURITY_POLICY_NAME_PREFIX):
-                        choices.append((policy_name, policy_name))
+            choices = self.conn.get_list('DescribeLoadBalancerPolicies', {}, [('member', Policy)])
```
Comment (dkavanagh) [inline eucaconsole/forms/__init__.py]: boto does not implement this call. I merely implemented it as boto would have.

**Pull request merged**